test/blackbox-tests/test-cases/default-targets/run.t

```diff
@@ -0,0 +1,15 @@
+Generates targets when modes is set for binaries:
+  $ dune build --root bins --display short @all 2>&1 | grep '\.bc\|\.exe'
+        ocamlc byteandnative.bc
+        ocamlc bytecodeonly.bc
+      ocamlopt nativeonly.exe
+      ocamlopt byteandnative.exe
+        ocamlc bytecodeonly.exe
+
+Generate targets when modes are set for libraries
+
+  $ dune build --root libs --display short @all 2>&1 | grep '\.cma\|\.cmxa\|\.cmxs'
+      ocamlopt byteandnative.cmxs
```

Comment:
Why aren't the .cmxa in here?

Reply:
My grep regex was broken. It's fixed now.